test/e2e/density.go

```diff
@@ -181,15 +187,13 @@ var _ = Describe("Density", func() {
 		// TODO: Expose runLatencyTest as ginkgo flag.
 		{podsPerNode: 3, runLatencyTest: false, interval: 10 * time.Second},
 		{podsPerNode: 30, runLatencyTest: true, interval: 10 * time.Second},
-		// More than 30 pods per node is outside our v1.0 goals.
-		// We might want to enable those tests in the future.
 		{podsPerNode: 50, runLatencyTest: false, interval: 10 * time.Second},
-		{podsPerNode: 100, runLatencyTest: false, interval: 1 * time.Second},
+		{podsPerNode: 100, runLatencyTest: true, interval: 10 * time.Second},
 	}
 
 	for _, testArg := range densityTests {
 		name := fmt.Sprintf("should allow starting %d pods per node", testArg.podsPerNode)
-		if testArg.podsPerNode == 30 {
+		if testArg.podsPerNode == 100 {
```

Comment:
I'm wondering what's the best way to do it. Note that we are running "Feature:Performance" suite also on larger clusters (e.g. 1000-node kubemark), where we don't want to run 100 pods/node density.

So personally, I would suggest setting some other tag for 100 pods/node, e.g.:
[Feature:PerformanceWithNode]   <- please suggest better name for it
and change our jenkins script to run tests with both tags in our 100-node scalability cluster.

@gmarek - thoughts?